2016/proposals/vora-meet.md

```diff
@@ -0,0 +1,209 @@
+* Show a working unit of test suite.
+
+## Future works
+Continue working on this project, along with AMY. I would continue to contribute, test and fix bugs that will be reported. I will work on improvements in AMY and also contribute to future projects of Software Carpentry.
+	
+## Open Source Development Experience
+I'm new to open source. This project would be my first opportunity to contribute to an open-source organization. I found the idea quite exciting - especially since I have already worked on few large-scale django projects. However, I have developed **Media Management Service**, a django-based backend service that handles media files and requests. It's features include media storage, file recovery, file transformation, file compression and user defined rule-based authentication. Further, it is integrated with Django Forms and [Django File Manager](https://github.com/IMGIITRoorkee/django-filemanager) (An open-source GUI File manager developed by Information Management Group). As soon as the documentation gets complete, I plan to release this service module as open source. 
+
+## Academic Experience
+I'm a second year undergraduate student at Indian Institute of Technology Roorkee (IIT R), majoring in Computer Science & Engineering. I have strong programming experience in Python, PHP, SQL and C++. As a part of academic curriculum, I have taken courses in *Software Engineering*, *Object Oriented Analysis & Design*, *Data Structures & Algorithms*, which have equipped me with skills to design, develop and test applications of industry standards.  
+Besides software development, I'm passionate about cryptography, learning theory and theoretical computer science.
+
+## Why this project?
+Being motivated to contribute to an open-source organization and having worked on Django for more than a year, I believe this project would be a perfect start to contribute to OSS. As a member of [Information Management Group](https://www.facebook.com/IMGIITRoorkee), I have worked on many django-based applications and have primarily worked on developing	Media Management Service and the Institute Placement Portal. The online competition portal that I developed for demonstration purpose can be found [here](https://github.com/meet-vora/euclid). This project seems very exciting to work on, given my background. Further, I have been able to use the SurveyMonkey API as well as Plotly during the discussion period and have well understood the idea. I have also realised the trouble one could get into, while using the SurveyMonkey API - querying with inclusion of custom variable *workshop_id* and OAuth, and have discussed the other details and issues [here](https://github.com/numfocus/gsoc/issues/98). Thus, I believe that I am well equipped with the skills, excitement and motivation to work on this project.  
+
+## Other Details
+#### Summer Schedule
+I have no other commitments scheduled for the summer and would devote atleast 50 hours a week, and more if required.
```

Comment:
I don't know if GSOC rules set a certain number of hours/week expected – but 50 hours/week is too much (unhealthy). From my experience as a GSOC student, it'd be more like 30+ hours (sometimes students are blocked by mentors).